decidim-admin/app/controllers/decidim/admin/concerns/has_content_blocks.rb

```diff
@@ -7,37 +7,16 @@ module HasContentBlocks
         extend ActiveSupport::Concern
 
         included do
-          include Decidim::Admin::LandingPage
-
-          helper_method :content_blocks?, :active_blocks, :inactive_blocks
-
-          def update_content_blocks
-            enforce_permission_to :update, :static_page, static_page: scoped_resource
-
-            Decidim::Admin::ReorderContentBlocks.call(current_organization, content_block_scope, params[:manifests], scoped_resource.id) do
-              on(:ok) do
-                head :ok
-              end
-              on(:invalid) do
-                head :bad_request
-              end
-            end
-          end
+          helper_method :content_blocks?
 
           private
 
-          def content_blocks?
-            return unless scoped_resource&.slug == "terms-and-conditions"
+          ACCEPTED_SLUGS = %w(terms-and-conditions).freeze
```

Comment:
I guess, that this should be actually a config setting like : 
```
Decidim.page_blocks = %w(terms-and-conditions).freeze
```
or even better 
```
Decidim::enable_page_blocks = true 
```
I remember while worked on futureeu.europa.eu, that a content block was required on another static page we had...

Reply:
I think we should go for the first option. The second option is a kind of all-or-nothing and the first one is more configurable.

Reply:
Applied!